src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -583,6 +584,20 @@ ReactDOMComponent.Mixin = {
         );
         didWarnShadyDOM = true;
       }
+
+      if (__DEV__) {
+        if (this._namespaceURI === DOMNamespaces.html) {
+          warning(
+            !(ownerDocument.createElement(this._tag) instanceof window.HTMLUnknownElement) ||
+              isCustomComponent(this._tag, props) || voidElementTags[this._tag],
```

Comment:
ok - totally agree - which means while using `menuitem` in chrome we should get the error message as expected. Fine but for that i will have to make some changes in the unit tests concerning `menuitem` to make them pass with this implementation.